src/java/main/org/apache/jute/compiler/JType.java

```diff
@@ -27,7 +27,7 @@
 	private String mCName;
     private String mCppName;
     private String mCsharpName;
-    private String mJavaName;
+    protected String mJavaName;
```

Comment:
Could you show me the findbug warning of ` protected String mJavaName` statement.